### PR TITLE
simplify mount function

### PR DESCRIPTION
**Pull request by kentcdodds**
If you look into what TestUtils is doing, this is basically it. The only reason you have to create a class component and use findDOMNode is because they don't return the `div` back to you. If you just do things this way then you don't need to bother with a class component.

I also renamed it to `element` because that's more accurate.

### CODE REVIEW

**Comment by sunesimonsen** [comment]
@kentcdodds that is pretty awesome, can I ask you to rebase it. I can also do it for you. Thanks for looking suggesting this!

**Comment by kentcdodds** [comment]
Fixed the conflict via github's editor. If you don't want the extra commit, just use `squash and merge` 👍

File: src/index.js [hunk 1]
```diff
@@ -2,20 +2,10 @@ import React, { Component } from "react";
...
-    TestUtils.renderIntoDocument(component) ||
-    TestUtils.renderIntoDocument(
-      <StatelessWrapper>{component}</StatelessWrapper>
-    );
-
-  return ReactDom.findDOMNode(rendered);
+export function mount(element) {
+  const div = document.createElement('div');
+  ReactDOM.render(element, div);
+  return div;
```
Comment (sunesimonsen) [inline src/index.js]: Don't we need to return `div.firstChild`?

Comment (sunesimonsen) [reply src/index.js]: That makes the tests pass.

**Comment by sunesimonsen** [comment]
@kentcdodds haha now it is a really tiny wrapper :-)

File: src/index.js [hunk 2]
```diff
@@ -2,20 +2,10 @@ import React, { Component } from "react";
...
-  const rendered =
-    TestUtils.renderIntoDocument(component) ||
-    TestUtils.renderIntoDocument(
-      <StatelessWrapper>{component}</StatelessWrapper>
-    );
-
-  return ReactDom.findDOMNode(rendered);
+export function mount(element) {
+  const div = document.createElement('div');
+  ReactDOM.render(element, div);
```
Comment (sunesimonsen) [inline src/index.js]: `ReactDOM` -> `ReactDom`, or rename the import.

Comment (kentcdodds) [reply src/index.js]: Let me know what you think about my other comment and I can update this.

File: src/index.js (hunk 1 above)
Comment (sunesimonsen) [reply src/index.js]: I'm sorry, but I would really like the mount function to just mount the element and return the mounted node. I think it is a bit confusing getting an extra `div` I didn't ask for. I would like any utility methods like the `queryByTestId` to be exported separately and take a node as the argument. We might have a bit different use-cases as I'm basically planing to use unexpected-dom on top of this library.

File: src/index.js (hunk 2 above)
Comment (kentcdodds) [reply src/index.js]: Fixed

File: src/index.js (hunk 1 above)
Comment (kentcdodds) [reply src/index.js]: Gotcha. I'm good with that. When I use this I can wrap it in a `div` myself if I want so that's fine :+1:

Thanks!

**Comment by sunesimonsen** [comment]
@kentcdodds thanks a bunch 👍 that really simplified things.

**Pull request merged**